tests/passing/dune

```diff
@@ -1,3 +1,12 @@
 (cram
+ (applies_to
+  (not windows-*))
+ (deps
+  (package ocp-indent)))
+
+(cram
+ (applies_to windows-*)
+ (enabled_if
+  (= %{os_type} "Windows"))
```

Comment:
You could make a test that runs under Windows and Unix indistinctly by creating a well-indented file with LF and checking that the output under `ocp-indent` is identical.

Reply:
Indeed, I was just changing it as my test was not run on the windows CI anyway.